aries-site/src/pages/templates/forms.mdx

```diff
@@ -23,29 +23,67 @@ import { Example } from '../../layouts';
   <SimpleSignUpExample />
 </Example>
 
-## Guidance
+### Jump to section...
+[Use cases and example code](#when-to-use)  
+[Do's and Don'ts visual examples](#dos)  
+[Use case and example code](use-cases)  
+&nbsp;&nbsp;&nbsp;&nbsp;[Sign up](#sign-up)  
+&nbsp;&nbsp;&nbsp;&nbsp;[Sign in](#sign-in)  
+&nbsp;&nbsp;&nbsp;&nbsp;[Filter](#filter)  
+&nbsp;&nbsp;&nbsp;&nbsp;[Sort](#sort)  
+&nbsp;&nbsp;&nbsp;&nbsp;[Change password](#change-password)   
+&nbsp;&nbsp;&nbsp;&nbsp;[Settings](#settings)  
+&nbsp;&nbsp;&nbsp;&nbsp;[Pay](#pay)  
+&nbsp;&nbsp;&nbsp;&nbsp;[Shipping](#shipping)   
+&nbsp;&nbsp;&nbsp;&nbsp;[Customize](#customize)  
+[Accessibility](#accessibility)   
+[Exceptions and how to address them](#exceptions-and-how-to-address-them)  
 
-Forms provide a way to collect important data from a user. However, this data is only collected effectively when Forms are built in a way in which users are inclined to fill out in full.
+## When to use
 
-### Usage
+To efficiently capture information necessary for completing a task.
 
-Forms don't need to be a tedious task for users to complete, as they often come to be. It is important to ease as much of this effort as possible when building a Form. Making Forms simple to complete leads to increase in form completion.
+## When not to use
 
-**Follow these guidelines when creating a Form:**
+Complicated processes where the user might need guidance at various stages, or when the form would be intimidatingly long. In these cases use a [Wizard](https://design-system.hpe.design/templates/wizard?q=error#guidance).
 
-- Use as few fields as possible
-- Wrap all input elements within a FormField
-- Fill in default values for as many fields as possible
-- Reveal fields progressively so the user is not overwhelmed
-- Use clear, concise labels for each field
-- Use placeholder text as an added visual indication of expected input value format
+## Do's
+
+- Ask only for the required information. Keep the fields to a minimum to allow the user to finish their task as quickly as possible.
```

Comment:
This is a very long list, is there any way we can break these into subcategories?

Reply:
Example would be...text field, label text, error text, filled text field,

Reply:
@taysea yes @bezAdane  mentioned that, too. Will be doing that very soon :)